tests/queryset_pickle/tests.py

```diff
@@ -115,6 +115,33 @@ def test_pickle_prefetch_related_idempotence(self):
         groups = pickle.loads(pickle.dumps(groups))
         self.assertSequenceEqual(groups, [g])
 
+    def test_pickle_prefetch_queryset_not_evaluated(self):
+        Group.objects.create(name='foo')
+        groups = Group.objects.prefetch_related(
+            models.Prefetch('event_set', queryset=Event.objects.order_by('id'))
+        )
+        list(groups)  # evaluate QuerySet
+        with self.assertNumQueries(0):
+            pickle.loads(pickle.dumps(groups))
+
+    def test_pickle_prefetch_queryset_usable_outside_of_prefetch(self):
+        # Prefetch shouldn't affect the fetch-on-pickle behaviour of the queryset passed to it
```

Comment:
Please use American spellings (behavior), periods, and wrap comments at 79 chars.